networkconfig/holesky.go

```diff
@@ -18,5 +18,7 @@ var Holesky = NetworkConfig{
 	Bootnodes: []string{
 		"enr:-Li4QFIQzamdvTxGJhvcXG_DFmCeyggSffDnllY5DiU47pd_K_1MRnSaJimWtfKJ-MD46jUX9TwgW5Jqe0t4pH41RYWGAYuFnlyth2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhCLdu_SJc2VjcDI1NmsxoQN4v-N9zFYwEqzGPBBX37q24QPFvAVUtokIo1fblIsmTIN0Y3CCE4uDdWRwgg-j",
 	},
-	WhitelistedOperatorKeys: []string{},
+	WhitelistedOperatorKeys:    []string{},
+	RSAForkEpoch:               220257, // Nov-27-2023 12:04:48 PM UTC
+	PermissionedDeactivateFork: 220257, // Nov-27-2023 12:04:48 PM UTC
```

Comment:
this epoch is in 3 years on Holesky :D

Nov 27 12:00 is at https://holesky.beaconcha.in/epoch/13500

Reply:
oh yeah, my bad I copied the same on from prater since it should be on the same time haha

Reply:
done